components/Dashboard/FormEdit.tsx

```diff
@@ -123,15 +125,21 @@ const FormEdit: React.FC<FormEditProps> = ({ form, onStepChange }) => {
               <FormControl>
                 <div className="relative mt-4 w-fit">
                   <input
+                    ref={inputref}
                     disabled={form.formState.isSubmitting}
                     className="w-[7.5rem] rounded bg-transparent px-1 py-1 text-xl font-bold outline-dashed outline-1 outline-transparent duration-100 hover:outline-gray-300 focus:outline-gray-300"
-                    {...field}
```

Comment:
add this `{...field}` before `ref={inputRef}` then it will not show that double initialization error

Reply:
```
<input
                    {...field}
                    disabled={form.formState.isSubmitting}
                    className="w-[7.5rem] rounded bg-transparent px-1 py-1 text-xl font-bold outline-dashed outline-1 outline-transparent duration-100 hover:outline-gray-300 focus:outline-gray-300"
                    ref={inputref}

```                    
like this

Reply:
Changes done as you have requested and I have commited the changes.